packages/ibm-products/src/components/CreateFullPage/CreateFullPageStep.tsx

```diff
@@ -82,7 +168,7 @@ export let CreateFullPageStep = forwardRef(
     }, [onMount, stepsContext, stepNumber, previousState?.currentStep]);
 
     useEffect(() => {
-      setShouldIncludeStep(includeStep);
+      setShouldIncludeStep(includeStep as any);
```

Comment:
Seems like `includeStep` is a boolean, could you please check this again and try to avoid type cast to 'any'?